Zend/tests/match/043.phpt

```diff
@@ -0,0 +1,37 @@
+--TEST--
+Match expression error messages
+--FILE--
+<?php
+
+class Beep {}
+
+function test(mixed $var) {
+    try {
+        match($var) {};
+    } catch (UnhandledMatchError $e) {
+        print $e->getMessage() . PHP_EOL;
+    }
+}
+
+test(1);
+test(5.5);
+test("foo");
+test(true);
+test(false);
+test([1, 2, 3]);
+test(new Beep());
+
+// Testing long strings.
+test(str_repeat('e', 100));
+test(str_repeat("e\n", 100));
+?>
+--EXPECT--
+Unhandled match 1
```

Comment:
I think there's something missing here ... possibly "Unhandled match on 1" or "Unhandled match of 1" or "Unhandled match on value 1" or something?

Reply:
Went for "case".